### DISCUSSION TITLE
Ram usage display

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#3276] [Ideas]
Can you please add ram usage. Have it below Gpu mode, with % used. Also bars would be cool, green to red.

**Comment by seerge** [comment]
@Reddot911 hello, line below GPU mode (next to display selection) is reserved for Mid Fan readings :)

RAM usage is a not very relevant parameter in modern windows, as system would just reserve a lot of RAM when it's not used and easily free it up when it's needed.

I might add it somewhere in the future, if I will be adding in-game overlay with some metrics. But in current main window - it's already cluttered enough.

**Reply by kwontemtek** [reply]
I have to agree with reddot911. I hate having to open up task manager to check my CPU usage % then open gHelper to check my thermals.  

In the spirit of keeping things simple I just just plain text like this could easily work:
44% CPU: 47C / 2800RPM | 50% RAM: 16GB/32GB 
15% iGPU: 48C | 0% dGPU: 46C / 2600RPM

**Comment by ghost** [comment]
Would be awesome to have, otherwise have to open task manager. Would be a great feature to have. I'm sure others would agree.

For example:
Move Visual mode / laptop keyboard into their own extra tab button.
Move down Laptop screeen hz and put ram usage there with a moving % bar green to red. 


![Screenshot 2024-10-21 214800](https://github.com/user-attachments/assets/7d4350b2-bbe9-470e-8b40-44719d06f858)

**Reply by seerge** [reply]
@Reddot911 hello,

Visual Mode block is relevant to display :) So showing RAM there would be not very logical.

What is the practical purpose of seeing free RAM ? modern windows manages memory dynamically, and reserves a lot of RAM when it's not used / frees it when apps need it. So actual RAM indicator often doesn't represent a real picture anyway.

Same as all this "memory optimizers" app now don't make any sense either

**Comment by ghost** [comment]
Any news on maybe getting this feature? would be aweeesooommeee
